src/test/kotlin/blackjack/BlackJackTest.kt

```diff
@@ -0,0 +1,41 @@
+package blackjack
+
+import blackjack.model.BlackJack
+import blackjack.model.Player
+import blackjack.model.Players
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class BlackJackTest {
+
+    @Test
+    fun `블랙잭 첫번째 턴 카드 뽑기`() {
+        val blackJack = BlackJack(Players(listOf(Player("a"), Player("b"))))
+        val a = blackJack.players.findPlayer(0)
+        val b = blackJack.players.findPlayer(1)
+        assertThat(a.cards.score() == 0).isEqualTo(true)
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        assertThat(a.cards.score()).isEqualTo(0)
```